e2e-tests/functions

```diff
@@ -10,7 +10,7 @@ MYSQL_VERSION=${MYSQL_VERSION:-"8.0"}
 IMAGE_PXC=${IMAGE_PXC:-"perconalab/percona-xtradb-cluster-operator:main-pxc${MYSQL_VERSION}"}
 IMAGE_PROXY=${IMAGE_PROXY:-"perconalab/percona-xtradb-cluster-operator:main-proxysql"}
 IMAGE_HAPROXY=${IMAGE_HAPROXY:-"perconalab/percona-xtradb-cluster-operator:main-haproxy"}
-IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-xtradb-cluster-operator:main-pxc${MYSQL_VERSION}-backup"}
+IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-xtradb-cluster-operator:k8spxc-1313-pitr"}
```

Comment:
We need to use the default backup image. If you add any changes to backup image please do not forget to create PR with these changes.

Reply:
Backup image doesn't have any changes but we build and use `pitr` executable in backup image's Dockerfile. That's why we need to use that image until this PR merged.